xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererDX.cpp

```diff
@@ -219,14 +219,14 @@ COverlayImageDX::COverlayImageDX(CDVDOverlayImage* o, CRect& rSource)
     m_x = (0.5f * o->width + o->x) / o->source_width;
     m_y = (0.5f * o->height + o->y) / o->source_height;
 
-    int videoSourceH{static_cast<int>(rSource.Height())};
-    int videoSourceW{static_cast<int>(rSource.Width())};
+    float subRatio{static_cast<float>(o->source_width) / o->source_height};
```

Comment:
Use of braces seems a bit strange here. Can you change to use `=`? 

And maybe use `const float` since this value is not expected to change once calculated.

Reply:
use braces because it is initialization not assignment
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#54-initialization

i will add const